skillmap/src/components/makecodeFrame.tsx

```diff
@@ -20,62 +20,69 @@ interface MakeCodeFrameProps {
     title: string;
     url: string;
     tutorialPath: string;
-    completed: boolean;
     activityHeaderId?: string;
     activityType: MapActivityType;
-    showCodeCarryoverModal: boolean;
-    carryoverModalVisible: boolean;
+    carryoverCode: boolean;
+    previousHeaderId?: string;
+    progress?: ActivityState;
     dispatchSetHeaderIdForActivity: (mapId: string, activityId: string, id: string, currentStep: number, maxSteps: number, isCompleted: boolean) => void;
     dispatchCloseActivity: (finished?: boolean) => void;
     dispatchSaveAndCloseActivity: () => void;
     dispatchUpdateUserCompletedTags: () => void;
-    dispatchShowCarryoverModal: (mapId: string, activityId: string) => void;
     dispatchSetReloadHeaderState: (state: "reload" | "reloading" | "active") => void;
```

Comment:
do we still need any of the reloadHeaderState flow now that the carryover code has moved?

Reply:
Nope! removed